update.sh

```diff
@@ -56,10 +56,16 @@ if [ $do_update ]; then
         fi
 
         # download the new package
-        downloadUrl="https://github.com/juwis/hackerschool/releases/download/$latestVersion/hucon.run"
+        downloadUrl="https://github.com/juwis/hackerschool/releases/download/$latestVersion/hucon-$latestVersion.run"
 
         # Download the new package
-        wget $downloadUrl
+        wget $downloadUrl -o hucon.run
+
+        echo "Check if existing code can be moved to /root/hucon/code..."
```

Comment:
What about the **examples** folder inside the code folder which will be copied?

Reply:
old examples will be deleted and the software only sources the new ones